dat/events/neutral/travelling_merchant.lua

```diff
@@ -294,7 +294,7 @@ They get uncomfortably close
       special_virtue    = outfit.get("Machiavellian Virtue"),
    }
    vn.label("special")
-   mm(fmt.f(_("So you're interested in my special services. Quite a bargain might I say. Each services costs only {cost}.")
+   mm(fmt.f(_("So you're interested in my special services. Quite a bargain might I say. Each services costs only {cost}."),
```

Comment:
Fixed a syntax error.

But travelling merchant will cause an error at vn.run() when you have some data matrices.